### ISSUE TITLE
Automatic formatting on stories written with stories using indented paragraphs breaks normal scene breaks.

### ISSUE THREAD

**Issue by dougnure** [knighty/fimfiction-issues#113]
With stories written in book-style formatting (Indented paragraphs with empty lines for scene breaks), the site's automatic formatting removes the extra lines, effectively hiding scene breaks. For all three formatting options, the existing scene break becomes indistinguishable from a new paragraph, unless an author goes back through all their stories using such formatting and specifically adds alternate scene-break cues.

**Comment by knighty** [comment]
This should not be the case. Can you give an example where it's happening? @dougnure

**Comment by TrickQuestion** [comment]
I think they might be referring to the fact that this:

> TAB   lorem ipsum lorem ipsum
> lorem ipsum lorem ipsum
> lorem ipsum
> TAB   lorem ipsum lorem ipsum
> lorem ipsum
> 
> TAB   lorem ipsum lorem ipsum 
> lorem ipsum

Now looks like:

> lorem ipsum lorem ipsum
> lorem ipsum lorem ipsum
> lorem ipsum
>
> lorem ipsum lorem ipsum
> lorem ipsum
> 
> lorem ipsum lorem ipsum 
> lorem ipsum

Notice that the break at the bottom looks just like the one at the top.

This would have been an issue for one of my stories, but I rewrote it recently to change the style. If this is what they are talking about, I don't think it's fixable except with the new tags. They'll need to reformat the stories.

**Comment by knighty** [comment]
Yeah but the thing is it /shouldn't/ do that. It should have the extra line break. I need to see an actual live example of it breaking.

edit: I misspoke. this is intended behaviour. The problem is that stories exist that use double spacing AND indenting to mark paragraphs and there is difficult to decide between for our automatic behaviour. We're looking at changing it to assume that double space and an indent correspond to a paragraph and line breaks, as the failure case for stories with both is merely extra line breaks rather than a loss of meaning from section breaks.

**Comment by dougnure** [comment]
Yes, it's as TrickQuestion described it. For one example, https://www.fimfiction.net/story/52953/without-a-hive

Written using book-style formatting, but scene breaks become invisible for all three formatting options.

**Comment by Fluttercheer** [comment]
Can confirm the same problem:

https://www.fimfiction.net/story/332551/1/aunt-millie/chapter-1-the-bad-news

_How it looks now:_

> **Chapter 1: The Bad News**
It was Monday in Ponyville, the beginning of a new week, and it was an ordinary morning for Dinky Hooves. The little unicorn sat at the table in the kitchen of her home, legs dangling down the chair, while she quietly hummed the newest hit of Sapphire Shores.

_Should be:_

> **Chapter 1: The Bad News**

> It was Monday in Ponyville, the beginning of a new week, and it was an ordinary morning for Dinky Hooves. The little unicorn sat at the table in the kitchen of her home, legs dangling down the chair, while she quietly hummed the newest hit of Sapphire Shores.

https://www.fimfiction.net/story/299751/3/the-strange-case-of-dinky-hooves-cutie-mark-curiosity/chapter-3-magical-misery-detour

_How it looks now:_

> “N-No!” Dinky answered hastily. “I really just need to clean it up! It will just be a moment, wait here until I'm done!” With that, she turned and rushed outside, leaving the Crusaders in the corridor, alone and confused.
> As the door had fallen shut behind her, Dinky darted over to the left side of the house. A few empty paint buckets stood there, left from the last time the house needed a repaint. Three long planks laid in the grass in front of the buckets and Dinky was grabbing one of them. She pulled it away from the house and over to the fence, where she let it drop again. She went back to the paint buckets, took them and arranged them on the plank.
>Dinky furrowed her brow, scrutinizing the arrangement, then shook her head. She turned around at the spot and galloped to the other side of the backyard, where she snuck around the corner of the house, aiming for the trash containers.....
> Meanwhile, Apple Bloom, Scootaloo and Sweetie Belle waited patiently inside the house.

_Should be:_

> “N-No!” Dinky answered hastily. “I really just need to clean it up! It will just be a moment, wait here until I'm done!” With that, she turned and rushed outside, leaving the Crusaders in the corridor, alone and confused.

> As the door had fallen shut behind her, Dinky darted over to the left side of the house. A few empty paint buckets stood there, left from the last time the house needed a repaint. Three long planks laid in the grass in front of the buckets and Dinky was grabbing one of them. She pulled it away from the house and over to the fence, where she let it drop again. She went back to the paint buckets, took them and arranged them on the plank.
> Dinky furrowed her brow, scrutinizing the arrangement, then shook her head. She turned around at the spot and galloped to the other side of the backyard, where she snuck around the corner of the house, aiming for the trash containers.....

> Meanwhile, Apple Bloom, Scootaloo and Sweetie Belle waited patiently inside the house.
